.travis.yml

```diff
@@ -1,16 +1,18 @@
 language: python
 
-python:
-    - "2.7"
+python: "2.7"
 
 sudo: false
 
-install: 
-    - "pip install -q -r requirements.txt --use-mirrors"
+cache:
+    directories:
+        - $HOME/.pip-cache/
+        - _build
+
+install: pip install --download-cache $HOME/.pip-cache --user sphinx==1.1.3
```

Comment:
``--user`` should not be used here. Python VMs are using a python virtualenv to isolate pip setups already (using ``--user`` is needed when using pip on non-Python VMs to avoid the need for sudo).